tests/functional/api/test_packages.py

```diff
@@ -148,3 +156,26 @@ def test_stream_generic_package_to_file(tmp_path, project):
 
     with open(path, "r") as f:
         assert f.read() == file_content
+
+
+def test_list_project_protected_packages(project: Project):
+    rules = project.package_protection_rules.list()
+    assert isinstance(rules, list)
```

Comment:
For now this just checks that we get a list as we can't programmatically create them yet (see below). Otherwise we'd get this:

```json
{
"message": "'packages_protected_packages' feature flag is disabled"
}
```